libdnf5/base/log_event.cpp

```diff
@@ -220,6 +256,13 @@ std::string LogEvent::to_string(
         case GoalProblem::WRITE_DEBUG:
             return ret.append(utils::sformat(_("Debug data written to \"{}\""), *additional_data.begin()));
         case GoalProblem::UNSUPPORTED_ACTION:
+            if (action == GoalAction::REVERT_COMPS_UPGRADE) {
+                return ret.append(utils::sformat(
+                    _("{} upgrade cannot be reverted, however associated package actions will be. ({} id: '{}') ."),
```

Comment:
May I ask you to mark an order of arguments `{1}`. It will allow to change order for translators. In this case what about to provide a context message for translators?